charts/prometheus-kafka-exporter/values.yaml

```diff
@@ -107,3 +107,9 @@ tls:
 # This is only needed for the CI job so it should always be disabled.
 kafka:
   enabled: false
+
+# Set security context for the kafka-exporter container. Useful when you need to adapt to an existing PSP
+securityContext: {}
+  # runAsUser: 
```

Comment:
@bogdan-matei can you please fix this lint issue?

Reply:
Fixed